### PR TITLE
Document component

### PR DESCRIPTION
**Pull request by teaaajo**
Added endpoints: Create new document, Get document by Id, Search Documents and postman collection json file + Tests

### CODE REVIEW

File: Levi9.ERP.Domain/Models/DTO/DocumentItemDTO.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿namespace Levi9.ERP.Domain.Models.DTO
+{
+    public class DocumentItemDTO
+    {
+        public int ProductId { get; set; }
+        public string Name { get; set; }
+        public float PriceValue { get; set; }
+        public string Currency { get; set; }
```
Comment (IvonaSkrinjar) [inline Levi9.ERP.Domain/Models/DTO/DocumentItemDTO.cs:8]: use enum for Currency

Comment (teaaajo) [reply Levi9.ERP.Domain/Models/DTO/DocumentItemDTO.cs:8]: Resolved

Comment (IvonaSkrinjar) [reply Levi9.ERP.Domain/Models/DTO/DocumentItemDTO.cs:8]: Cannot find enum CurrencyType in DocumentItemDTO

Comment (IvonaSkrinjar) [inline Levi9.ERP.Domain/Models/DTO/DocumentItemDTO.cs:8]: enum CurrencyType already exists in the project. Use it instead of string

**Review by IvonaSkrinjar — APPROVED** [review]

**Pull request merged**